core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1457,4 +1457,14 @@ private SqlLibraryOperators() {
           OperandTypes.family(SqlTypeFamily.ANY,
               SqlTypeFamily.STRING),
           SqlFunctionCategory.TIMEDATE);
+
+  @LibraryOperator(libraries = {BIG_QUERY, HIVE, SPARK, SNOWFLAKE})
+  public static final SqlFunction TRANSLATE =
```

Comment:
Please add a test for the TRANSLATE function in calcite.

Reply:
Done